src/git.md

```diff
@@ -434,6 +434,5 @@ exist and that they correspond to some sort of embedded subrepository dependency
 that Git can nicely and fairly conveniently handle for us.
 
 [Git submodules]: https://git-scm.com/book/en/v2/Git-Tools-Submodules
-[`rust-toolstate`]: https://rust-lang-nursery.github.io/rust-toolstate/
```

Comment:
how did you notice this is unused... do you have a tool

Reply:
I just noticed while searching for rust-toolstate. I would like to have a tool to detect those automatically. Unfortunately I don't think pulldown-cmark provides quite enough information for that.